### ISSUE TITLE
visitors keep dying.

### ISSUE THREAD

**Issue by AndrewTru** [ldtteam/minecolonies#6341]
### Prerequisites 

- [x] I am running the latest **alpha** version of MineColonies and Structurize for my Minecraft version.
- [X] I checked the MineColonies/Structurize [wiki](https://wiki.minecolonies.ldtteam.com/) and made sure my issue is not covered there.
- [X] I made sure that **this issue is not a duplicate of any existing issue.**

### Context 

- Minecraft Version: 1.16.4
- MineColonies Version: 0.13.547-ALPHA
- Structurize Version: 0.13.104-ALPHA
- Related Mods and their Versions:
JEI: 7.6.1.63, JER: 0.12.0.103, light overlay: 5.6.1, Cloth API: 4.1.3
Architectury: 1.2.68

## Expected behavior

tavern calls in visitors, they hang out for a few days, leave.

## Actual behavior

A visitor is made, find them in the wall, they take damage, die.
Not sure if it's based on the level but any level after 1 I think.
the first update messes with things that cover areas and aren't clearing old landing zones.

## Steps to reproduce the issue

1. start a colony.
2. build a tavern.
3. wait and see if you have guests.

### Logs

- latest.log: 
- crashlog: 

### Notes

---
## Viewers

* Add a thumbs-up to the bug report if you are also affected. This helps the bug report become more visible to the team and doesn't clutter the comments.
* Add a comment if you have any insights or background information that isn't already part of the conversation.

**Comment by kmarple1** [comment]
I had this same issue with the Asian level 1 tavern. Upgrading it resolved the issue, so it might be related to the schematic? They also liked to die in the campfire that's part of that schematic.

**Comment by AndrewTru** [comment]
Okay, but this is a level 2 I think, and it's still happened.
not sure why?

**Comment by camrex** [comment]
With the latest update I am getting the same issue with Visitors dying. (All in the same location.) I have a jungle village, with a level 2 tavern.

**Comment by PasixTheValkyrie** [comment]
I don't have this issue with a wooden tavern

**Comment by ravenbuilder934** [comment]
Can anyone having this issue please try repairing their tavern and check if that fixes it?

**Comment by camrex** [comment]
I believe my Jungle Tavern is currently being upgraded to a level 3, which should be equivalent to a repair. I will check to see if that fixed it when I get home this evening, once the upgrade is complete.

**Comment by LeviFarstad** [comment]
I have a Level 2 Fortress tavern that is experiencing this issue. I've tried repairing it, but that hasn't resolved the issue, am about to try upgrading it. I've also noticed that my visitors are sitting outside, which makes me think the "Sit here" area isn't placed correctly and I could see that causing visitors to "sit" inside a wall. Actually managed to get an image of what would have been a wall death (was just trying to show that visitors were sitting).

![2021-01-05_11 13 00](https://user-images.githubusercontent.com/77018090/103677686-388df880-4f48-11eb-9e82-c56bbe9d3b7b.png)
![2021-01-05_11 13 55](https://user-images.githubusercontent.com/77018090/103677690-39bf2580-4f48-11eb-9a2e-3183b73a27a4.png)

Then as I was typing this up I heard the "Death ticks" and captured these two pictures in order:

![2021-01-05_11 22 43](https://user-images.githubusercontent.com/77018090/103677853-712dd200-4f48-11eb-938d-c1319db4bd08.png)
![2021-01-05_11 22 55](https://user-images.githubusercontent.com/77018090/103677854-712dd200-4f48-11eb-8d49-fc1cf58d2cc1.png)

*I had a previous colony on the same save with the same tavern type that did not experience this issue. 
*The old safe colony was on land, the new one with the wall deaths is on the ocean. 
*All images are at level 2 with one repair attempt completed
*MineColonies version: minecolonies-0.13.547-ALPHA-universal
*Structurize version: structurize-0.13.104-ALPHA-universal

Edit: Updating to level 3 did not fix the issue, all behavior above is still happening

**Comment by AndrewTru** [comment]
wow, so not just me then.
wow....can someone test the other themes? to see if it's in others.

**Comment by malhavok13** [comment]
I built mine in fortress style. I never had an issue until I initiated an upgrade. Then my visitors started sitting outside on nothing, then started dying in the walls. Upgraded again to lvl 3 and still having same issues.

**Comment by matthew-williams** [comment]
Same issue. It was fine till I upgrade to the level 2 Fortress version

**Comment by matthew-williams** [comment]
![image](https://user-images.githubusercontent.com/6619443/104133531-6bb5fa80-5341-11eb-84fd-9d280d0aa3c6.png)

**Comment by matthew-williams** [comment]
![Screenshot 2021-01-10 191604](https://user-images.githubusercontent.com/6619443/104142506-2c0b0500-5379-11eb-85de-8d1c32374203.png)
You are off by about 5 blocks to the West. For the spawning area in both the Level 3 and the Level 2 Schematic.

**Comment by matthew-williams** [comment]
Actually looks like 5 to the south as well. So 5 too the West and 5 to the south.

**Comment by michmerr** [comment]
I'm playing on a server with a modpack that's about a month old. MC 0.13.547-ALPHA, ST 0.13.104-ALPHA.

At least some visitors from my level 2 birch tavern are spawning (I assume) about 40 block from the tavern door. I assume this because they are drowning in the two-story aquariums of the level 3 birch hospital next door. Both aquariums. They are sealed on all sides.

This is my first play through, so I was figuring a lot of stuff out at the stage when I was placing the tavern. It's possible that I shifted it around a lot before planting it and/or starting the build. Other than seeing if a repair would fix the situation (it did not), I haven't done anything unusual with it since it was built.

**Comment by michmerr** [comment]
My latest visitor drowning in a sealed aquarium was a visitor that had been walking around the colony the previous day. So, it wasn't a new arrival spawn, which is what I had been assuming before. So, if there are events where a visitor could clip through a wall (spawns, chunk loading, whatever), that might be where they're getting rounded into the next block.

**Comment by Raycoms** [comment]
fixed

**Comment by nomnomaproved** [comment]
minecolonies 0.13.596 alpha tavern walls killing ppl

**Comment by ravenbuilder934** [comment]
Not fixed then

**Comment by Raycoms** [comment]
If your build is still in progress (from before the fix) or things are wrongly tagged this can still happen. Then it's #5006 though.
